src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -14,13 +14,34 @@ import PrintButton from '../../components/popUp/PrintButton';
 import { FiltersChange } from '../../components/popUp/types';
 import { SummaryFiltersUpdate } from '../../highlights/types';
 import { LinkedArchiveTreeNode } from '../../types';
+import { splitTitleParts } from '../../utils/archiveTreeUtils';
 import { printStudyGuides, updateSummaryFilters } from '../actions';
 import { highlightStyles } from '../constants';
 import * as selectors from '../selectors';
 import { cookieUTG } from './UsingThisGuide/constants';
 import UsingThisGuideBanner from './UsingThisGuide/UsingThisGuideBanner';
 import UsingThisGuideButton from './UsingThisGuide/UsingThisGuideButton';
 
+const createColorDataAnalyticsLabel = (color: HighlightColorEnum): string => {
```

Comment:
It would be good to add a comment here explaining that this is created based on current values from `message.json`